module/Entra/AdditionalFunctions/Get-EntraAdministrativeUnit.ps1

```diff
@@ -0,0 +1,139 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+function Get-EntraAdministrativeUnit {
+    [CmdletBinding(DefaultParameterSetName = 'GetQuery')]
+    param (
+    [Parameter(ParameterSetName = "GetById", Mandatory = $true, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.String] $ObjectId,
+    [Parameter(ParameterSetName = "GetQuery", ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.Nullable`1[System.Int32]] $Top,
+    [Parameter(ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [switch] $All,
+    [Parameter(ParameterSetName = "GetQuery", ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.String] $Filter
+    )
+
+    PROCESS {    
+    $params = @{}
+    $customHeaders = New-EntraCustomHeaders -Command $MyInvocation.MyCommand
+    $baseUri = "/v1.0/directory/administrativeUnits"
+    $properties = '$select=*'
+    $params["Uri"] = "$baseUri/?$properties"
+    if($null -ne $PSBoundParameters["ErrorAction"])
+    {
+        $params["ErrorAction"] = $PSBoundParameters["ErrorAction"]
+    }
+    if($null -ne $PSBoundParameters["ObjectId"])
+    {
+        $params["AdministrativeUnitId"] = $PSBoundParameters["ObjectId"]
+        $params["Uri"] = "$baseUri/$($params.AdministrativeUnitId)?$properties"
+    }
+    if($PSBoundParameters.ContainsKey("Verbose"))
```

Comment:
Remove Common parameters since we are invoking API directly

Reply:
removed and updated